rest/src/main/java/discord4j/rest/util/DiscordUtils.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file is part of Discord4J.
+ *
+ * Discord4J is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Discord4J is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Discord4J.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package discord4j.rest.util;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+public final class DiscordUtils {
```

Comment:
Ah, the resurgence of DiscordUtils of v2.

I think the naming can be changed a bit. For one, the other "utility" classes use `Util` instead of `Utils`, so I think this should follow. Additionally, I don't really like incredibly generalized utility classes (or naming to suggest as such). I propose `TokenUtil` instead, for potentially in the future being able to do other utility things based off tokens (maybe something can be done with Bearer as well?).